eng/build.yml

```diff
@@ -15,13 +15,13 @@ parameters:
   timeoutInMinutes: 180
 
 jobs:
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/7.0.4xx'), not(contains(parameters.agentOs, 'TemplateEngine'))) }}:
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranch'], 'refs/heads/release/8.0.1xx'), not(contains(parameters.agentOs, 'TemplateEngine'))) }}:
```

Comment:
The branch you targeting is `refs/heads/release/7.0.4xx`. Why change it to `8.0.1xx`?

Reply:
We only localize a single branch at a time. I switched the branch to 8.0.1xx on the loc side but without this change we'll still be running the onelocbuild in the 7.0.4xx branch. Potentially, I could have just deleted this entire section but by updating to 8.0.1xx, it'll flow smoothly in inter-branch codeflow (it's already been changed in the 8.0.1xx branch)